cloudify_cli/cli/helptexts.py

```diff
@@ -206,17 +206,6 @@
        'This argument can be used multiple times'
 PERMISSION = 'The permission applicable to a resource [viewer|owner] ' \
              '(default:viewer)'
-RESTORE_SNAPSHOT_TENANT_NAME = 'The name of the tenant into which the ' \
```

Comment:
I thought we were removing the entire thing? There is no helptext with it removed.